llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -6727,6 +6756,11 @@ bool VPCostContext::skipCostComputation(Instruction *UI, bool IsVector) const {
          SkipCostComputation.contains(UI);
 }
 
+unsigned VPCostContext::getPredBlockCostDivisor(
```

Comment:
Is this needed when the caller could just use `Ctx.CM.getPredBlockCostDivisor`?

Reply:
`getPredBlockCostDivisor` is only declared in LoopVectorize.cpp so it's not visible from VPlanRecipes.cpp etc. 

I think a few of VPCostContext's methods exist just to forward calls to the legacy CostModel e.g. VPCostContext::isLegacyUniformAfterVectorization